src/modules/blockchainApplication/explore/components/BlockchainApplicationList/BlockchainApplicationList.js

```diff
@@ -58,7 +59,8 @@ const BlockchainApplicationList = () => {
         <QueryTable
           showHeader
           queryHook={useBlockchainApplicationExplore}
-          queryConfig={{ config: { params: filters } }}
+          queryConfig={{ config: { params: { ...filters } } }}
```

Comment:
is there are reason for the extra spread here I think its additional operation that is not needed since QueryTable isn't going to be mutating params

```suggestion
          queryConfig={{ config: { params: filters } }}
```

Reply:
@eniolam1000752 It was accidentally left after some changes to that line. Fixed